WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -948,6 +949,15 @@ public void onEventMainThread(FollowedTagsFetched event) {
                 updateCurrentTag();
             }
         }
+
+        // Check last time we've bumped tags followed analytics for this user,
+        // and bumping again if > 1 hrs
+        long tagsUpdatedTimestamp = AppPrefs.getReaderAnalyticsCountTagsTimestamp();
+        long now = new DateProvider().getCurrentDate().getTime();
+        if (now - tagsUpdatedTimestamp > 1000 * 60 * 60) { // 1 hr
+            ReaderTracker.trackFollowedTagsCount(ReaderTagTable.getFollowedTags().size());
+            AppPrefs.setReaderAnalyticsCountTagsTimestamp(now);
+        }
```

Comment:
💡 Would it make sense to move this logic to `ReaderPostListViewModel` so we can unit test this and keep this logic away from the View? The `ViewModel` even has an instance of the `ReaderTracker` injected so we could even avoid the static call.